### PR TITLE
Update knuddels-userapps-api

### PR DESCRIPTION
**Pull request by SirezKN**
Please fill in this template.

- [x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [x ] Test the change in your own code. (Compile and run.)
- [x ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **types/knuddels-userapps-api/index.d.ts#L14** - The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d2ca40e1d431f917be2b6e138712ed4b798075c1
</p>

**Review by Trve — APPROVED** [review]

File: types/knuddels-userapps-api/index.d.ts [hunk 1]
```diff
@@ -1902,12 +1916,12 @@ declare global {
...
 			user: User,
 			callback: (user: User, bonusEndTimestamp: number, bonuses: [{
-		productId: string,
-		origPriceCents: number,
-		newPriceCents?: number,
-		origKnuddelPayout: number,
-		newKnuddelPayout?: number
-	}], result: string) => void
+				productId: string,
+				origPriceCents: number,
```
Comment (sandersn) [inline types/knuddels-userapps-api/index.d.ts:1920]: please use spaces instead of tabs

**Review by sandersn — CHANGES_REQUESTED** [review]

**Review by Trve — APPROVED** [review]
everything ok.

**Comment by SirezKN** [comment]
Ready to merge

**Pull request merged**